### ISSUE TITLE
startup.py tests close the loggers

### ISSUE THREAD

**Issue by remram44** [VisTrails/VisTrails#1024]
```
  vistrails/core/startup.py(549)test_config_override()
-> self.close_logger()
```

**Comment by dakoop** [comment]
I think I was unable to clear away a newly created .vistrails directory used to test startup without first making sure the loggers weren't writing to files in that directory. Perhaps something needs to be reenabled after killing off the loggers?

**Comment by remram44** [comment]
I see. It seems that creating a `VistrailsStartup` will call `DebugPrint#set_logfile()`, opening a new logfile.
I'm just going to mock it.

**Comment by remram44** [comment]
Fixed by 22b0e981.
